corehq/apps/export/utils.py

```diff
@@ -188,3 +192,17 @@ def _convert_index_to_path_nodes(index):
         return path
     else:
         return [PathNode(name=n) for n in index.split('.')]
+
+
+def revert_new_exports(new_exports):
```

Comment:
This could use a docstring.  Looks like it's intended for when the feature flag was enabled then later disabled?